specs/Freeze.md

```diff
@@ -0,0 +1,155 @@
+used by the renderer process for other processes.   
+Note that the freeze is best effort and considered completed successfully once the request
+is sent to browser renderer process. If there is a running script, the script will continue
+to run and the renderer process will be frozen after that script is done.
+See [Sleeping Tabs FAQ](https://techcommunity.microsoft.com/t5/articles/sleeping-tabs-faq/m-p/1705434)
+for conditions that might prevent WebView from being frozen. In those situations,
+The TryFreeze operation will fail and the completed handler will be invoked with isSuccessful as false.   
+The WebView will be automatically unfrozen when it becomes visible. Therefore, the app normally doesn't have to call Unfreeze.
+The app can call `Unfreeze` and then `TryFreeze` periodically for an invisibile WebView so that the invisible WebView can sync up with
+latest data and the page ready to show fresh content when it becomes visible.
+
+# API Notes
+See [API Details](#api-details) section below for API reference.
+
+# API Details
+
+## Win32 C++
+```IDL
+interface ICoreWebView2_2 : ICoreWebView2 {
+
+  /// An app may call the `TryFeezeAsync` API to have the WebView2 consume less memory.
+  /// This is useful when a Win32 app becomes invisible, or when a Universal Windows
+  /// Platform app is suspended, during the suspended event handler before completing
+  /// the suspended event.
+  /// The CoreWebView2Controller must be invisible when the API is called. Otherwise, the
+  /// API fails with `HRESULT_FROM_WIN32(ERROR_INVALID_STATE)`.
+  /// Freezing is similar to putting a tab to sleep in the Edge browser. Freezing pauses
+  /// WebView script timers and animations, minimizes CPU usage for the associated
+  /// browser renderer process and allows the operating system to reuse the memory that was
+  /// used by the renderer process for other processes.
+  /// Note that the freeze is best effort and considered completed successfully once the request
+  /// is sent to browser renderer process. If there is a running script, the script will continue
+  /// to run and the renderer process will be frozen after that script is done.
+  /// See [Sleeping Tabs FAQ](https://techcommunity.microsoft.com/t5/articles/sleeping-tabs-faq/m-p/1705434)
+  /// for conditions that might prevent WebView from being frozen. In those situations,
+  /// The TryFreeze operation will fail and the completed handler will be invoked with isSuccessful as false.
+  /// The WebView will be automatically unfrozen when it becomes visible. Therefore, the
+  /// app normally doesn't have to call Unfreeze.
+  /// The app can call `Unfreeze` and then `TryFreeze` periodically for an invisibile WebView so that
+  /// the invisible WebView can sync up with latest data and the page ready to show fresh content
+  /// when it becomes visible.
+  HRESULT TryFreeze([in] ICoreWebView2StagingTryFreezeCompletedHandler* handler);
```

Comment:
(Separating out comments from previous PR https://github.com/MicrosoftEdge/WebView2Feedback/pull/621/files#r533302500)

Is it acceptable to have a Try method with no corresponding non-Try method?

This seems fine by [WinRT rules](https://osgwiki.com/wiki/WinRT_API_Design_Patterns#Try_Pattern) is it for .NET? For both WinRT & .NET there seems to be plenty of precedent of Try methods with no corresponding non-Try method.

Reply:
A Try without a non-Try is fine. Guideline though is to have sync versions of async methods.

Reply:
Though I don't think the guideline really applies to UI methods, since most UI methods require access to the UI thread, which won't work if it's blocked on a sync UI method.

Reply:
That was going to be our next question =)

Reply:
Not providing sync version as all webview APIs happen on UI thread.